build/builder_unix.go

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//go:build !windows
 // +build !windows
```

Comment:
unrelated: I'm guessing go fmt added this for you. But we only support go1.17+, so no need to keep the old format around.

```suggestion
//go:build !windows
```